packages/cli/src/Server.ts

```diff
@@ -2595,6 +2595,20 @@ class App {
 			);
 		}
 
+		if (process.env.NODE_ENV !== 'production') {
+			ignoredEndpoints.push('dev-node-description-updated');
```

Comment:
1. I would add this at the initialization of `ignoredEndpoints` so we have a single place where to find ignored endpoints.
2. Trying to hit from Insomnia, I get `Unauthorized` even though I have the cookie. Are you able to hit this manually?